lib/gui/pages/settings/settings.js

```diff
@@ -25,7 +25,8 @@ const MODULE_NAME = 'Etcher.Pages.Settings';
 const SettingsPage = angular.module(MODULE_NAME, [
   require('angular-ui-router'),
   require('../../components/warning-modal/warning-modal'),
-  require('../../models/settings')
+  require('../../models/settings'),
+  require('../../modules/error')
```

Comment:
Hmmm? How did the code work without this `require('../../modules/error')` before?